CHANGELOG.md

```diff
@@ -4,6 +4,68 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.14.0] - 2023-04-06
+
+## 🚀 Features
+
+### GraphOS Enterprise: Coprocessor read access to request `uri`, `method` and HTTP response status codes ([Issue #2861](https://github.com/apollographql/router/issues/2861), [Issue #2861](https://github.com/apollographql/router/issues/2862))
+
+We've added the ability for [coprocessors](https://www.apollographql.com/docs/router/customizations/coprocessor) to have read-only access to additional contextual information at [the `RouterService` and `SubgraphService`](https://www.apollographql.com/docs/router/customizations/coprocessor/#how-it-works) stages:
+
+The `RouterService` stage now has read-only access to the **request** from the client:
+  - `path` (e.g., `/graphql`)
+  - `method` (e.g., `POST`, `GET`)
+
+The `RouterService` stage now has read-only access to the overall **response** to the client:
+  - `status_code` (e.g. `403`, `200`)
+
+The `SubgraphService` stage now has read-only access to the **response** of the subgraph request:
+  - `status_code` (e.g., `503`, `200`)
```

Comment:
The read only access only applies to the path / method / status_code, and it reads like the whole request and response are read only.

Reply:
@o0Ignition0o Good call.  Does this help?

```suggestion
The `RouterService` stage now has read-only access to these **client request** properties:
  - `path` (e.g., `/graphql`)
  - `method` (e.g., `POST`, `GET`)

The `RouterService` stage now has read-only access to these **client response** properties:
  - `status_code` (e.g. `403`, `200`)

The `SubgraphService` stage now has read-only access to these **subgraph response** properties:
  - `status_code` (e.g., `503`, `200`)
```

Reply:
LGTM!